DOCS/man/osc.rst

```diff
@@ -400,6 +400,12 @@ Configurable Options
     Use ``no`` to disable chapter display on hover. Otherwise it's a lua
     ``string.format`` template and ``%s`` is replaced with the actual name.
 
+``unicodeminus``
+    Default: no
+
+    Use a Unicode minus sign instead of an ASCII hyphen when displaying
+    negative values.
```

Comment:
> when displaying negative values.

Please change to something like "... ASCII hyphen at the time-remaining duration". I don't think there are other negative values at the OSC, and if there are then this commit does not modify them.